### ISSUE TITLE
Unable to build on ArchLinux

### ISSUE THREAD

**Issue by tyhawk** [vankasteelj/opensubtitles-uploader#99]
I'm trying to build opensubtitles-downloader on ArchLinux. Tried both the PKGBUILD in the AUR (https://aur.archlinux.org/packages/opensubtitles-uploader) as well as manually from source, but I am running into a problem. Building (with NPM) hangs right at the start. Strangely enough, I was able to build the current version without problems about 2 months ago.
```
(⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂) ⠼ idealTree:opensubtitles-uploader-2.6.0: sill idealTree buildDeps
```
I wonder what the issue is, wether a certain NPM package (version) is unable to be fetched or another issue. I'm not excluding an issue with dependent package versions Arch is using, and I am open to suggestions to better troubleshoot this issue.

So far, I found this in the logs:
```
24 timing npm:load:logFile Completed in 7ms
25 timing npm:load:timers Completed in 0ms
26 timing npm:load:configScope Completed in 0ms
27 timing npm:load Completed in 79ms
28 timing arborist:ctor Completed in 1ms
29 timing arborist:ctor Completed in 1ms
30 timing idealTree:init Completed in 31ms
31 timing idealTree:userRequests Completed in 0ms
32 silly idealTree buildDeps
33 silly fetch manifest detect-lang@^1.2.0
```
detect-lang is also written by @vankasteelj and as far as I can see there is nothing wrong with the availability of the package (https://www.npmjs.com/package/detect-lang). I've also excluded DNS query blocking within my network; requests to registry.npmjs.org are not blocked and permitted. It's also not blocked by a firewall (local or networkwide).

**Comment by vankasteelj** [comment]
Hi, you might be using an older or newer version of npm. Locally I use npm 6.14.15 and node 14.18.0 if it helps.

**Comment by Edoarda77** [comment]
Dear Sir,
I completely ignore what you are writing to me about.  I don't have informatics knowledge, beyond what is absolutely essential to use a computer.  I am sorry that you are wasting so much of your time on trying to explain things and procedures to me.  You see, I am now 82 and it is impossible to me to learn these procedures.

Sorry that you are wasting your time on writing to me and apologize for this.

Best regards
Edoarda Paolini

**Comment by tyhawk** [comment]
Haven't been able to build up to now. Current versions in ArchLinux are NPM 8.15.1 and NodeJS 18.7.0.
Unfortunately, I lack the programming skills to troubleshoot this.

The binary work, so I'll make a PKGBUILD for ArchLinux users so they can still use this program even if it can't be compiled at this point in time.
